communication/zoom-guidelines.md

```diff
@@ -0,0 +1,39 @@
+# Zoom Guidelines
+
+Zoom is the main video communication platform for Kubernetes outside of our mailing lists. 
+It is used for running the [community meeting](https://github.com/kubernetes/community/blob/master/events/community-meeting.md) and SIG meetings. 
+Since the Zoom meetings are open to the general public, a Zoom host might have to moderate a meeting if a person is in violation of the code of conduct. 
+
+These guidelines are meant as a tool to help Kubernetes members manage their Zoom resources. 
+Check the main [moderation](./moderation.md) page for more information on other tools and general moderation guidelines.
+
+## Code of Conduct
+Kubernetes adheres to Cloud Native Compute Foundation's [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) throughout the project, and includes all communication mediums.
+
+## Moderation
+
+Zoom has documentation on how to use their moderation tools: 
+
+- https://support.zoom.us/hc/en-us/articles/201362603-Host-Controls-in-a-Meeting
+
+Check the "Screen Share Controls" (via the ^ next to Share Screen): Select who can share in your meeting and if you want only the host or any participant to be able to start a new share when someone is sharing.  
+
+You can also put an attendee on hold. This allows the host(s) to put attendee on hold to temporarily remove an attendee from the meeting. 
+
+Unfortunately, Zoom doesn't have the ability to ban or block people from joining - especially if they have the invitation to that channel and the meeting id is publicly known.
+
+It is strongly encouraged that at least two people who lead SIG Meetings are comfortable with how to use these moderation tools. 
```

Comment:
IMO - Required that the host be comfortable and strongly encouraged at least two...